decentraland/explorer-desktop
# fix: report bug button overlapping with private chat icons

## What does this PR change?

Report bugs button has been moved too the bottom right of the screen.

## How to test the changes?



1. Click the report bug button
2. Should open a new browser tab to open a new github issue

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
